devbuild.sh

```diff
@@ -237,6 +245,42 @@ if [ "${VERBOSE}" = true ] ; then
   settings
 fi
 
+# Check out external components ===============================================
+if [ "${APPLICATION}" = "ATMAQ" ]; then
+  if [ -d "${SRW_DIR}/sorc/arl_nexus" ]; then
+    printf "!!! Extra external components already exist. This step is skipped.\n"
+  else
+    if [ -d "${SRW_DIR}/sorc/gsi" ]; then
```

Comment:
This `if` test is not sufficient to produce the desired behavior.  What happens when `${SRW_DIR}/sorc/gsi` does not exist, but one some permutation of `${SRW_DIR}/sorc/ufs-weather-model`, `${SRW_DIR}/sorc/UFS_UTILS`, `${SRW_DIR}/sorc/UPP`, `${SRW_DIR}/sorc/rrfs_utl` does exist?  I think the test needs work to ensure that the correct behavior is rendered when any combination of things that shouldn't exist (and needs to be removed), does exist.

Reply:
I get what your point is. I'll change this if-statement soon.

Reply:
@christopherwharrop-noaa, I've removed this if-statement, so the 'manage_externals/checkout_externals' call should be done before running the 'devbuild.sh' script.